maskit/examples/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0"
```

Comment:
```suggestion
```

The submodule does not require any version – so you can easily remove this line. This will even simplify our lives as we would not have to remember changing the version string each time we are bumping the version. I also don't even know how the tools used for publication would handle this...